### DISCUSSION TITLE
考没考虑做一个图形界面？

### DISCUSSION THREAD

**Discussion by SongZihui-sudo** [liuyunfz/chaoxing_tool#33] [Ideas]
考没考虑做一个图形界面？

**Comment by liuyunfz** [comment]
当然可以啊，如果可以我还想把代码重构一遍呢，我这基本用的全是面向过程，函数一多可阅读性就很差，耦合性太高了。但综合来说还是懒，现在抽时间出来修复bug就已经花费我的全部精力了，否则对不起大家的这么多star。

**Reply by SongZihui-sudo** [reply]
可以用窗口做前端来显示，直接把函数调用先绑定到按钮上，chaoxing.py也可以先不用重构。。。。。。

**Reply by liuyunfz** [reply]
如果您愿意进行相关的开发的话那么十分感谢，我本人可能暂时不会参与开发了。生活中的琐事较为繁忙，也没有特别的驱动力让我能下决心来挤出时间。目前的基本bug维护就是我所能做的全部了

**Reply by SongZihui-sudo** [reply]
哦哦，好的

**Reply by SongZihui-sudo** [reply]
![图片](https://user-images.githubusercontent.com/77034643/166691159-10fffc3f-1b91-4e69-900c-56cdfe3970cd.png)    

用tkinter简单弄一下，多少有点简陋。

**Reply by 1457764171** [reply]
> ![图片](https://user-images.githubusercontent.com/77034643/166691159-10fffc3f-1b91-4e69-900c-56cdfe3970cd.png)
> 
> 用tkinter简单弄一下，多少有点简陋。

可以把做好的页面，分享一下吗

**Reply by SongZihui-sudo** [reply]
https://github.com/liuyunfz/chaoxing_tool/tree/master/desktop 这

**Reply by 1457764171** [reply]
代码运行错误，加不了桌面版的，成品的话部分课程选择之后按end结束不了这个好像没办法使用
